sdk.yaml

```diff
@@ -288,17 +288,18 @@ Realtime:
   Options:
     .synopsis: |
       `ClientOptions` that only apply to Realtime clients.
-    Automatic Connection:
-      .specification: [RTC1b, RTN3, RTL4b1, TO3e]
-      .synopsis: |
-        `autoConnect`:
-        Disable the automatic initiation of a connection when a client instance is instantiated.
     Channel Retry Timeout:
       .specification: [RTL13b, TO3l7]
       .synopsis: |
         `channelRetryTimeout`:
         Channel reattachement is attempted after this amount of time in the `SUSPENDED` connection state,
         following a server-initiated channel detach, if the connection state is `CONNECTED`.
+    Connection Lifecycle Control:
+      .specification: [RTC1b, RTN3, RTL4b1, TO3e, RTC15, RTC16, RTN11, RTN12]
+      .synopsis: |
+        `autoConnect`:
+        Disable the automatic initiation of a connection when a client instance is instantiated.
+        Implies the presence of `connect` and `close` methods on client instances, allowing the application to control when the connection process is started, as well explicitly requesting connection closure.
```

Comment:
I agree this implies `connect`, but does it imply `close`? I think `close` is needed regardless of whether `autoConnect` is true or false.

Reply:
I guess the point I was trying to make is that all three, grouped together, provide _explicit_ control from the app-side of things around the lifecycle of the connection (opening _and_ closing). I can't imagine a scenario where we might implement `close()` but not `connect()`.

Maybe the problem is my use of the word 'implies'?

Reply:
Ok makes sense, I do think it's the word "implies" that didn't click with me, perhaps "assumes", "expects", "requires"?

Reply:
I've gone with "requires": https://github.com/ably/features/pull/20/commits/d6c0dd9a401a5204c1c0b36ac8e63675eb1cf518

Reply:
I'm going to resolve this conversation on @lmars' behalf in order to get this pull request merged.